src/vm/nativefieldcategory.h

```diff
@@ -0,0 +1,29 @@
+enum NativeFieldSubcategory
+{
+    NATIVE_FIELD_CATEGORY_FLOAT = 1 << 10,
+    NATIVE_FIELD_SUBCATEGORY_NESTED = 1 << 11,
+#ifdef FEATURE_COMINTEROP
```

Comment:
I don't see any reason to do this for an enum value. You can remove the `#ifdef`.